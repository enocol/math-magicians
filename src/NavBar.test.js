import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from './components/NavBar';

test('Navbar component snapshot', () => {
  const { asFragment } = render(
    <Router>
      <Navbar />
    </Router>,
  );
  expect(asFragment()).toMatchSnapshot();
});
