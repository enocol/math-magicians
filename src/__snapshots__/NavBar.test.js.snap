// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component snapshot 1`] = `
<DocumentFragment>
  <div
    class="navbar-container"
  >
    <nav
      class="navbar"
    >
      <div>
        <span
          class="logo"
        >
          Math Magicians
        </span>
      </div>
      <ul
        class="navbar-links"
      >
        <li>
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li
          class="border"
        >
          <a
            href="/calculator"
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            href="/quotes"
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </div>
</DocumentFragment>
`;
